### PR TITLE
Instant60 VIA Support

### PR DESCRIPTION
**Pull request by awkannan**
## Description

Add VIA support to Instant60 and 
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cannonkeys/instant60/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#define _BASE 0
+#define _FN1 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT_all(
+    KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,                 KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSLS, KC_DEL, \
```
Comment (fauxpark) [inline keyboards/cannonkeys/instant60/keymaps/via/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

File: keyboards/cannonkeys/instant60/keymaps/tsangan/keymap.c [hunk 2]
```diff
@@ -31,18 +31,18 @@ enum custom_keycodes {
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   [_BASE] = LAYOUT_tsangan(
-    KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,                 KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSPC, KC_DEL, \
```
Comment (fauxpark) [inline keyboards/cannonkeys/instant60/keymaps/tsangan/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

**Comment by awkannan** [comment]
I got Wilba's blessing

File: keyboards/cannonkeys/instant60/info.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "Instant60",
+    "url": "https://cannonkeys.com",
+    "maintainer": "CannonKeys",
```
Comment (noroadsleft) [inline keyboards/cannonkeys/instant60/info.json]: `maintainer` should actually be your GitHub username.

```suggestion
    "maintainer": "awkannan",
```

File: keyboards/cannonkeys/practice65/info.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Practice65",
+    "url": "https://cannonkeys.com",
+    "maintainer": "CannonKeys",
```
Comment (noroadsleft) [inline keyboards/cannonkeys/practice65/info.json]:
```suggestion
    "maintainer": "awkannan",
```

File: keyboards/cannonkeys/instant60/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/cannonkeys/instant60/keymaps/via/keymap.c]:
```suggestion
```

File: keyboards/cannonkeys/instant60/keymaps/via_standard/keymap.c [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/cannonkeys/instant60/keymaps/via_standard/keymap.c]:
```suggestion
```

File: keyboards/cannonkeys/instant60/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
```
Comment (fauxpark) [inline keyboards/cannonkeys/instant60/keymaps/via/keymap.c]: This keycode doesn't appear to be used.

File: keyboards/cannonkeys/instant60/keymaps/via_standard/keymap.c [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
```
Comment (fauxpark) [inline keyboards/cannonkeys/instant60/keymaps/via_standard/keymap.c]: And same here.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**